src/main/java/core/basesyntax/CustomMathPow.java

```diff
@@ -5,10 +5,18 @@
     /**
      * Написать метод pow(int number, int exponential), который является аналогом метода
      * Math.pow() и принимает 2 целых числа:
-     *      number - число которое надо возводить в какую-то степень
-     *      exponential - значение степени
+     * number - число которое надо возводить в какую-то степень
+     * exponential - значение степени
      */
     public int pow(int number, int exponential) {
-        return 0;
+        double value = 1;
+        if (number != 0) {
```

Comment:
@Rosinskyi
Зроби краще навпаки:
```suggestion
        if (number == 0) {
```